### DISCUSSION TITLE
v4.2.0

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5015] [Announcements]
v4.2.0

VS Code v1.64.2

### Added

- Added tests for `handleArgsSocketCatchError`, `setDefaults` and
  `optionDescriptions`.

### Changed

- We switched from using the fork `coder/vscode` to a submodule of
  `microsoft/vscode` + patches managed by `quilt` for how Code sits inside the
  code-server codebase.
- Upgraded to Code 1.64.2.

### Fixed

- Update popup notification through `--disable-update-check` is now fixed.
- Fixed PWA icons not loading on iPad
- Fixed the homebrew release process. Our `cdrci` bot should now automatically
  update the version as part of the release pipeline.
- Fixed titleBar color setting being ignored in PWA.

### Security

- Updated to `minimist-list`.
- Updated `cloud-agent` to `v0.2.4` which uses `nhooyr.io/webscoket` `v1.8.7`.

## New Contributors
* @azriel-stephen made their first contribution in https://github.com/coder/code-server/pull/4981

**Full Changelog**: https://github.com/coder/code-server/compare/v4.1.0...v4.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/coder/code-server/releases/tag/v4.2.0'>v4.2.0</a>.</em>

**Comment by Matthew-Beckett** [comment]
_nice._

<img src="https://media1.giphy.com/media/KPGLbBdxRbehi/giphy.gif"/>
